.github/workflows/issue_event_moves_card.yml

```diff
@@ -0,0 +1,121 @@
+            echo "PVII_id is $PVII_id"
+            if [ "$issue_id" == "$ISSUE_NODE_ID" ]; then
+              # Extract the status field id
+              status_field_id=$(echo $project_fields_and_issues | jq -r '.data.node.fields.nodes[] | select(.name == "Status") | .id')
+              # Get the current status of the issue
+              current_status=$(echo $project_fields_and_issues | jq -r --arg PVII_id "$PVII_id" '.data.node.items.nodes[] | select(.id == $PVII_id) | .fieldValues.nodes[] | select(.field.name == "Status") | .name')
+              echo "current_status is $current_status"
+              priority_message=""
+              workload_message=""
+              status_message=""
+              if [ "$current_status" == "$CURRENT_COLUMN_NAME" ]; then
+                # Move the issue to the specified column
+                Option_id=$(echo $project_fields_and_issues | jq -r --arg COLUMN_NAME "$COLUMN_NAME" '.data.node.fields.nodes[] | select(.name == "Status") | .options[] | select(.name == $COLUMN_NAME) | .id')
+                echo "Option_id is $Option_id"
+                echo "change field $status_field_id of issue $issue_id in project $project_id to $COLUMN_NAME"
+                status_message="updateProjectV2ItemFieldValue: updateProjectV2ItemFieldValue(input: {projectId: \"'$project_id'\", itemId: \"'$PVII_id'\", fieldId: \"'$status_field_id'\", value: { singleSelectOptionId: \"'$Option_id'\"}}) {projectV2Item {id}}"
+                echo "status_message is $status_message"
+              fi
+              if [ "$PRIORITY" != 'NONE' ]; then
+                priority_field_id=$(echo $project_fields_and_issues | jq -r '.data.node.fields.nodes[] | select(.name == "Priority") | .id')
+                priority_option_id=$(echo $project_fields_and_issues | jq -r --arg PRIORITY "$PRIORITY" '.data.node.fields.nodes[] | select(.name == "Priority") | .options[] | select(.name == $PRIORITY) | .id')
+                priority_message="updateProjectV2ItemFieldValue1: updateProjectV2ItemFieldValue(input: {projectId: \"'$project_id'\", itemId: \"'$PVII_id'\", fieldId: \"'$priority_field_id'\", value: { singleSelectOptionId: \"'$priority_option_id'\"}}) {projectV2Item {id}}"
+                echo "priority_message is $priority_message"
+              fi
+              if [ "$WORKLOAD" != 'NONE' ]; then
+                workload_field_id=$(echo $project_fields_and_issues | jq -r '.data.node.fields.nodes[] | select(.name == "Workload") | .id')
+                workload_option_id=$(echo $project_fields_and_issues | jq -r --arg WORKLOAD "$WORKLOAD" '.data.node.fields.nodes[] | select(.name == "Workload") | .options[] | select(.name == $WORKLOAD) | .id')
+                workload_message="updateProjectV2ItemFieldValue2: updateProjectV2ItemFieldValue(input: {projectId: \"'$project_id'\", itemId: \"'$PVII_id'\", fieldId: \"'$workload_field_id'\", value: { singleSelectOptionId: \"'$workload_option_id'\"}}) {projectV2Item {id}}"
```

Comment:
Not important, but it would be cool to have a more readable message somewhere, like above: Workload changed to ...

Reply:
yes, but this is to check that the queries are constructed correctly. I'll add a more comprehensible output.